### PR TITLE
Add docs for Inflector::dasherized()

### PR DESCRIPTION
**Pull request by ionas**
Ref: https://github.com/cakephp/docs/issues/3654
Also ordered methods by alphabet 
Thanks @lorenzo for pointing me to the API instead of the book.

### CODE REVIEW

File: en/core-libraries/inflector.rst [hunk 1]
```diff
@@ -80,6 +81,9 @@ table names or class names::
     // UserProfileSetting
     Inflector::classify('user_profile_settings');
 
+    // user-profile-settings
+    Inflector::dasherize('UserProfileSetting');
```
Comment (dereuromark) [inline en/core-libraries/inflector.rst]: Does dasherize make it plural? UserProfileSetting => user-profile-settings

**Comment by ionas** [comment]
Maybe that example should be put in a different paragraph. Where?

File: en/core-libraries/inflector.rst [hunk 2]
```diff
@@ -80,6 +81,9 @@ table names or class names::
     // UserProfileSetting
     Inflector::classify('user_profile_settings');
```
Comment (dereuromark) [inline en/core-libraries/inflector.rst]: This looks still off though.

Comment (markstory) [inline en/core-libraries/inflector.rst]: I can fix that up post merge.

Comment (ionas) [inline en/core-libraries/inflector.rst]: I don't think I changed Classify... thats from the old docs. If that changed from 2.x it should be in the migration docs?

Comment (dereuromark) [inline en/core-libraries/inflector.rst]: @ionas : @markstory already understood me and fixed it post merge. all done.

**Pull request merged**